plugin-gestures/src/main/java/com/mapbox/maps/plugin/gestures/GesturesPluginImpl.kt

```diff
@@ -1075,7 +1074,7 @@ class GesturesPluginImpl : GesturesPlugin, GesturesSettingsBase, MapStyleObserve
     detector: ShoveGestureDetector
   ) {
     // re-enabling move gesture
```

Comment:
imho comments are no needed or could move to gestureState

Reply:
Let's remove the old comments